src/components/portal/vendor/VendorTypeAhead.js

```diff
@@ -19,14 +19,13 @@ class VendorTypeAhead extends Component {
                 vendor: '',
                 vendorList: [],
                 cur_id: '',
-                valid_vendor: false
+                valid_vendor: false,
```

Comment:
This comma probably doesn't break anything because JS is hard to break, but could we remove it just for cleanliness?